FWCore/Integration/test/unit_test_outputs/testSubProcess.grep.txt

```diff
@@ -1,6 +1,14 @@
 Sharing ESSource: class=DoodadESSource label=''
 Sharing ESSource: class=DoodadESSource label=''
 Sharing ESSource: class=DoodadESSource label=''
+++++++++ starting: prefetching for esmodule: label = '' type = DoodadESSource in record = GadgetRcd
```

Comment:
Why is the ESModule label empty here?

Reply:
You can see it is also empty on the line above. We've always had it such that if the name associated to the `cms.Process` for an ES module is identical to that ES module's C++ class name than the label is blank. That stems from the days before we used python as a configuration language where only the class name was needed to specify the loading of an ES module.